wiki/People/The_Team/Accounts_management_team/en.md

```diff
@@ -0,0 +1,31 @@
+# Account management team
+
+*For a team that moderates certain subforums, see: [Support Team](/wiki/People/The_Team/Support_Team)*
+
+**The account management team** is a group of people administering user accounts, as well as accepting silence and restriction appeals.
+
+## Point of contact
+
+The team can **only** be reached by sending a mail to [support@ppy.sh](mailto:support@ppy.sh) or [accounts@ppy.sh](mailto:accounts@ppy.sh). The identities of its members are kept secret in order to protect them from unwanted attention.
+
+## Role and tasks
+
+*Main article: [Help Centre](/wiki/Help_Centre)*
+
+*See also: [Account Restrictions](/wiki/Help_Centre/Account_Restrictions)*
+
+You should contact this team regarding account-related topics that are out of your control. Please remember to provide as much context on your issue as possible and send a email from the address associated with your osu! account.
+
+### [accounts@ppy.sh](mailto:accounts@ppy.sh)
+
+- [Appeals of restrictions placed on your account](/wiki/Help_Centre/Account_Restrictions), including silences and profile content removals.
+- Instances of misbehaviour that you [wish to admit](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
+- [Lose of access to the email](/wiki/Help_Centre#sign-in) tied to your osu! account, or when your account is stolen.
+- Username reverts and [minor spelling corrections](/wiki/Help_Centre#name-changes).
```

Comment:
```suggestion
- [Username reverts and minor spelling corrections](/wiki/Help_Centre#name-changes).
```
I see no point in not covering the rest under the hyperlink

Reply:
it's poor style when the whole sentence is a link. I'll leave the first word out as a compromise